org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WPAttributeDialog.java

```diff
@@ -215,13 +218,22 @@ private void init () throws Exception
 		mainLayout.setParent(this);
 		ZKUpdateUtil.setHflex(mainLayout, "1");
 		ZKUpdateUtil.setVflex(mainLayout, "min");
+		mainLayout.setStyle("max-height: 600px;");

```

Comment:
This wouldn't works well on mobile with smaller height. one option is to do it with condition, i.e `if (ClientInfo.maxHeight(600) mainLayout.setStyle("max-height: 100%;"); else mainLayout.setStyle("max-height: 600px;");`

As a safety net, we can setMaxizable(true) for the dialog. This give user a way to make the dialog usable if there's problem with default sizing set by the application.